src/decode.c

```diff
@@ -819,6 +819,24 @@ const char *PacketDropReasonToString(enum PacketDropReason r)
     return NULL;
 }
 
+/** \brief Decide Packet's final verdict based on packet action, return it as a string */
+const char *PacketActionVerdictToString(const Packet *p)
+{
+    /* Reject is valid in both IDS and IPS */
+    if (PacketCheckAction(p, ACTION_REJECT_ANY)) {
+        return "reject";
```

Comment:
I'd like to rethink the reject here. I feel it takes something very simple (was a packet accepted or dropped) and makes it complicated and hard to reason about. Perhaps we need to split it out. E.g.

IPS:
```json
"verdict": {
     "verdict": "drop"
     "active-response": "reject"
}
```
and IDS:
```json
"verdict": {
     "active-response": "reject"
}
```
(I don't like the verdict.verdict here, need a better name. But not enough coffee yet to come up with one)

Reply:
I think just as much information is provided with

IPS:

```
"verdict": "drop"
```

IDS:

```
"verdict": "reject"
```

Reply:
I disagree. Is a packet dropped as well as rejected in IPS mode? What about IDS mode?

Reply:
Lets enumerate the actions, what do we have:
- drop
- reject
- drop-and-reject
Anything else?

Maybe a new noun is needed:
- verdict: the decision made, this could simply be allow/deny
- action: this is the action taken.. drop, reject, both, ?

Reply:
Pondering on this  to try to come up with a suggestion...

Reply:
I think we have two aspects to consider:
- the initial problem - having only `action` leading to ambiguous results (when, say, a packet triggers signatures with conflicting/different actions and therefore `action` isn't informative enough of the final action taken on the packet). 
- And: that `reject` has different results in `IPS` and `IDS` and, moreover, that in `IPS` `reject` also means dropping the packet.

The `verdict` field seems to only be insufficient in `reject`'s case, when I can't see how to solve this with a single word. And even then, given reject's implications, I think this still should require going to the documentation.

I think that we should avoid using the `action` noun, considering we use it for the signatures. 

However I look at this, I end up feeling that we'll be redundant and lack a bit of info, at the same time, when it comes to `reject`... >_<

Reply:
How about:
```json
"verdict": {
   "action": "reject",
   "ips-verdict": "drop",
   "active-response": {
       "name": "tcp-reset"
       "direction": "toserver"
   }
}
```

Reply:
I think we need to provide example rules, and modes that lead to what the output would look like, as I do find it a bit hard to reason about.

If we take a court as an example, verdict is indepedent of action.  The verdict might be guilty, not-guilty.  Action could be nothing, jail, jail+fine, etc.

So in the example above, I believe verdict is `reject` (from the rule), and the action is drop+reject. So I think `ips-verdict` doesn't have the base name.  Yes, the verdict was drop, but did we drop it?

Reply:
Do you think that we should change the words used, or that the documentation/ rule examples/ SV tests showcasing could be enough?